packages/product-react/src/Tabs/TabView.scss

```diff
@@ -0,0 +1,3 @@
+.ray-product__tab-view:focus {
+  outline: none;
```

Comment:
I want to call this out to @danaballasy and @simon that we're removing the blue outline of the container. I know that there are very strong opinions on keeping it due to accessibility, but since this would be for the container below the tab, I suggested that we remove it.

Reply:
Yeah we need an answer on this one

Reply:
my opinion is that we should not be removing it. there is still a need for users to be able to visibly detect the body of the tab is focused